GoogleUtilities.podspec

```diff
@@ -78,7 +78,9 @@ other Google CocoaPods. They're not intended for direct public usage.
 
   s.subspec 'ISASwizzler' do |iss|
     iss.source_files = 'GoogleUtilities/ISASwizzler/**/*.[mh]', 'GoogleUtilities/Common/*.h'
+    iss.public_header_files = 'GoogleUtilities/ISASwizzler/Private/*.h'
     iss.private_header_files = 'GoogleUtilities/ISASwizzler/Private/*.h'
+    iss.requires_arc = ['GoogleUtilities/Common/*.h', 'GoogleUtilities/ISASwizzler/GULObjectSwizzler*.[mh]']
```

Comment:
It seems scary to fix an issue by disabling ARC. Would you explain the reasoning?

Reply:
I tried to explain it at the [file with disabled ARC](https://github.com/firebase/firebase-ios-sdk/pull/3300/files#diff-c53e9d63fddb6fe3b2d071f7a8f529b5R66) itself, but probably I need to add more detailed explanation. 
The core idea was expressed by Jonathan [here](https://groups.google.com/a/google.com/d/msg/appledev-discuss/vtQkgbxK9Vk/Pey3kRb4CAAJ). Basically, we use MRC here to define the required order  of the object deinitialization.